test/scales/scaleTests.ts

```diff
@@ -94,21 +84,29 @@ describe("Scales", () => {
       svg2.remove();
     });
 
-    it("scale perspectives can be removed appropriately", () => {
-      assert.isTrue((<any> scale)._autoDomainAutomatically, "autoDomain enabled1");
-      scale._updateExtent("1", "x", d3.extent(data, (e) => e.foo));
-      scale._updateExtent("2", "x", d3.extent(data, (e) => e.bar));
-      assert.isTrue((<any> scale)._autoDomainAutomatically, "autoDomain enabled2");
-      assert.deepEqual(scale.domain(), [-20, 5], "scale domain includes both perspectives");
-      assert.isTrue((<any> scale)._autoDomainAutomatically, "autoDomain enabled3");
-      scale._removeExtent("1", "x");
-      assert.isTrue((<any> scale)._autoDomainAutomatically, "autoDomain enabled4");
-      assert.closeTo(scale.domain()[0], -20, 0.1, "only the bar accessor is active");
-      assert.closeTo(scale.domain()[1], 1, 0.1, "only the bar accessor is active");
-      scale._updateExtent("2", "x", d3.extent(data, (e) => e.foo));
-      assert.isTrue((<any> scale)._autoDomainAutomatically, "autoDomain enabled5");
-      assert.closeTo(scale.domain()[0], 0, 0.1, "the bar accessor was overwritten");
-      assert.closeTo(scale.domain()[1], 5, 0.1, "the bar accessor was overwritten");
+    it("addExtentProvider()", () => {
+      var posProvider: Plottable.Scales.ExtentProvider<number> = (scale: Plottable.Scale<number, number>) => [[0, 10]];
+      scale.addExtentProvider(posProvider);
```

Comment:
Can we inline the Provider here?  I don't really see the purpose of separating into two lines

```typescript
scale.addExtentProvider(() => [[0, 10]]);
```